lib/datahub-client/data_platform_catalogue/entities.py

```diff
@@ -14,6 +15,94 @@ class RelationshipType(Enum):
     CHILD = "CHILD"
 
 
+class DatahubEntityType(Enum):
+    CONTAINER = "CONTAINER"
+    DATASET = "DATASET"
+    DASHBOARD = "DASHBOARD"
+    CHART = "CHART"
+    GLOSSARY_TERM = "GLOSSARY_TERM"
+
+
+@dataclass
+class FindMoJDataEntityMapper:
+    find_moj_data_type: str
+    datahub_type: str
+    datahub_subtypes: list[str]
+    url_formatted: str
+
+
+class TableEntityMapper(FindMoJDataEntityMapper):
+    def __init__(self):
+        super().__init__(
+            "Table",
+            DatahubEntityType.DATASET.value,
+            ["Model", "Table", "Seed", "Source"],
+            "table"
+        )
+
+
+class ChartEntityMapper(FindMoJDataEntityMapper):
+    def __init__(self):
+        super().__init__("Chart", DatahubEntityType.CHART.value, [], "chart")
+
+
+class DatabaseEntityMapper(FindMoJDataEntityMapper):
+    def __init__(self):
+        super().__init__("Database", DatahubEntityType.CONTAINER.value, ["Database"], "database")
+
+
+class DashboardEntityMapper(FindMoJDataEntityMapper):
+    def __init__(self):
+        super().__init__("Dashboard", DatahubEntityType.DASHBOARD.value, [], "dashboard")
+
+
+class PublicationDatasetEntityMapper(FindMoJDataEntityMapper):
+    def __init__(self):
+        super().__init__(
+            "Publication dataset",
+            DatahubEntityType.DATASET.value,
+            ["Publication dataset"],
+            "publication_dataset"
+        )
+
+
+class PublicationCollectionEntityMapper(FindMoJDataEntityMapper):
+    def __init__(self):
+        super().__init__(
+            "Publication collection",
+            DatahubEntityType.CONTAINER.value,
+            ["Publication collection"],
+            "publication_collection"
+        )
+
+
+class EntityTypes(Enum):
```

Comment:
This is redundant with FindMoJDataEntityMapper now - something to address in the refactor ticket?

Both have the Find MoJ data entity type, Datahub entity type, and URL formatted, but FindMoJDataEntityMapper also has the Datahub subtypes.

Reply:
That class I created, but it's not used anywhere. kinda my bad. We can remove it or leave it with the intention of working it in to the rest of the code